radio/src/inactivity_timer.cpp

```diff
@@ -56,8 +56,12 @@ bool inactivityCheckInputs()
 {
   uint8_t sum = 0;
 
-  for (uint8_t i = 0; i < adcGetMaxInputs(ADC_INPUT_ALL); i++)
-    sum += anaIn(i) >> INAC_STICKS_SHIFT;
+  auto max_inputs = adcGetMaxInputs(ADC_INPUT_MAIN)
+    + adcGetMaxInputs(ADC_INPUT_FLEX);
```

Comment:
You can use the offset of the next category instead (it's stored in FLASH).
Something like:
```c++
auto max_inputs = adcGetInputOffset(ADC_INPUT_FLEX + 1);
```

Reply:
I think that makes the code harder to understand.

Reply:
True, no problem with me.